src/CourseAuthoringRoutes.test.jsx

```diff
@@ -75,54 +63,51 @@ describe('<CourseAuthoringRoutes>', () => {
 
   fit('renders the PagesAndResources component when the pages and resources route is active', () => {
     render(
-      <AppProvider store={store} wrapWithRouter={false}>
-        <MemoryRouter initialEntries={['/pages-and-resources']}>
-          <CourseAuthoringRoutes />
-        </MemoryRouter>
-      </AppProvider>,
-    );
-
-    expect(screen.getByText(pagesAndResourcesMockText)).toBeVisible();
-    expect(mockComponentFn).toHaveBeenCalledWith(
-      expect.objectContaining({
-        courseId,
-      }),
+      <MemoryRouter initialEntries={['/pages-and-resources']}>
+        <CourseAuthoringRoutes />
+      </MemoryRouter>,
     );
+    waitFor(() => {
```

Comment:
You can simplify this since our `render()` in testUtils already provides a `MemoryRouter`:
```suggestion
      <CourseAuthoringRoutes />,
      { routerProps: { initialEntries: ['/pages-and-resources'] } },
    );
    await waitFor(() => {
```
Also this test is broken without `await`

Reply:
Added, thanks